### DISCUSSION TITLE
Has anyone tried to use MapScaler together wit Altair?

### DISCUSSION THREAD

**Discussion by knutwannheden** [vega/altair#2726] [Q&A] [answered]
An example for how to use MapScaler (https://pypi.org/project/mapscaler/) together with Altair's `mark_geoshape()` would be most appreciated!

**Comment by mattijn** [comment, marked as answer]
Mapscaler just updates your geometry. So this should work:
```python
import mapscaler as ms

# mapscaler stuff
loader = ms.MapLoader()
df = loader.fetch_counties()['df']
ss = ms.ShapeScaler()
scaled_df = ss.scale_map(df, 'EST_POP_2019', map_vel=.001, group_vel=.15, verbose=True)

import altair as alt
alt.Chart(scaled_df).mark_geoshape().project('albersUsa')
```

But for me the mapscaler function runs more than 30 minutes. So the problem is not so much altair in this.

**Reply by knutwannheden** [reply]
Thanks for the example! The runtime overhead is a bummer. I will give it a try.
